_gsocprojects/2020/project_Ceph.md

```diff
@@ -0,0 +1,12 @@
+---
+project: Ceph
+layout: default
+logo: ceph-logo.png
+description: |
+  [Ceph](https://ceph.io/) is a storage platform.  It implements object storage on a single distributed computer cluster, and provides interfaces for object-, block- and file-level storage. It can be used to build cloud infrastructure and web-scale object storage. Ceph aims primarily for completely distributed operation without a single point of failure, that can be run on commodity hardware and is scalable to the exabyte level.
+summary: |

```

Comment:
summary: is not required (description: is used by default) , but recommended. It only makes sense when (summary < description). Either reduce summary or remove.